### PR TITLE
Edwardkoval rsa2019 new provider topic page

### PR DESCRIPTION
**Pull request by edwardkoval**

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[4c962cf](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/4c962cfea63ed0cc05f7b56136a7349a705daca5)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 26 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[4c962cf](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/4c962cfea63ed0cc05f7b56136a7349a705daca5)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 4 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[4c962cf](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/4c962cfea63ed0cc05f7b56136a7349a705daca5)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/security-partner-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/edwardkoval-RSA2019-NewTopicPage/concepts/security-partner-overview.md) | :white_check_mark:Succeeded | |
[concepts/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/edwardkoval-RSA2019-NewTopicPage/concepts/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C2%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903020147320003-4074%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ivSVCFAxP2msx8pgERs28S2zQTBjzQ%2BqUqSyiVNEFsQ%3D&st=2019-03-02T01%3A46%3A21Z&se=2019-04-02T01%3A51%3A21Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[68b275b](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/68b275bb8a2face2fd55f37bf2410a3d92cbd56b)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 26 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[68b275b](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/68b275bb8a2face2fd55f37bf2410a3d92cbd56b)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 4 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[68b275b](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/68b275bb8a2face2fd55f37bf2410a3d92cbd56b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/security-partner-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/edwardkoval-RSA2019-NewTopicPage/concepts/security-partner-overview.md) | :white_check_mark:Succeeded | |
[concepts/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/edwardkoval-RSA2019-NewTopicPage/concepts/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C2%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903020152168089-4074%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=MCBBfAQZsNEIlJdI2uX6eTP11DgjfoSHF2sU4CJ%2BQGE%3D&st=2019-03-02T01%3A51%3A06Z&se=2019-04-02T01%3A56%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[ecdde00](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/ecdde0002f003de58fd48a723c03d55c490ac10b)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 26 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[ecdde00](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/ecdde0002f003de58fd48a723c03d55c490ac10b)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 4 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[ecdde00](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/ecdde0002f003de58fd48a723c03d55c490ac10b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/security-partner-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/edwardkoval-RSA2019-NewTopicPage/concepts/security-partner-overview.md) | :white_check_mark:Succeeded | |
[concepts/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/edwardkoval-RSA2019-NewTopicPage/concepts/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C2%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903020156276948-4074%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=RT2NJxLRV9wl9B2B7HW3fBRq3Y5ekoP9R1s8YacOAhA%3D&st=2019-03-02T01%3A54%3A55Z&se=2019-04-02T01%3A59%3A55Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[14bc68b](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/14bc68bbcee9beb1fc5b7081b06d2375a5931131)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 26 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[14bc68b](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/14bc68bbcee9beb1fc5b7081b06d2375a5931131)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 4 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[14bc68b](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/14bc68bbcee9beb1fc5b7081b06d2375a5931131)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/security-partner-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/edwardkoval-RSA2019-NewTopicPage/concepts/security-partner-overview.md) | :white_check_mark:Succeeded | |
[concepts/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/edwardkoval-RSA2019-NewTopicPage/concepts/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C2%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903020159338062-4074%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=4cbE1r6kpD%2FRLJVMn%2BdOdAqLYPSBagAHC4tEQ5YFyYA%3D&st=2019-03-02T01%3A58%3A35Z&se=2019-04-02T02%3A03%3A35Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[1c8fe43](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/1c8fe43674f0b9411e23980050d9338217396595)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 26 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[1c8fe43](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/1c8fe43674f0b9411e23980050d9338217396595)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 4 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[1c8fe43](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/1c8fe43674f0b9411e23980050d9338217396595)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/security-partner-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/edwardkoval-RSA2019-NewTopicPage/concepts/security-partner-overview.md) | :white_check_mark:Succeeded | |
[concepts/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/edwardkoval-RSA2019-NewTopicPage/concepts/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C2%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903020205057017-4074%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=EZW%2BW03Gr%2F1D4rRnUNyrahnxRcatAaccYTYLQTUeJI8%3D&st=2019-03-02T02%3A03%3A32Z&se=2019-04-02T02%3A08%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[8141368](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/814136800c8ba8f031bc1391a223b9db4d6d9f1f)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 26 sec(s)_

File: concepts/security-partner-overview.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+---
+title: "Partnering with Microsoft Graph Security – technology partner opportunities "
+description: "This document is an introduction to partnering opportunities enabled by Microsoft Graph Security and is designed to help product managers and business development roles understand the investment paths and provide insight into partnering value propositions. "
+localization_priority: Normal
+author: "preetikr"
+ms.prod: "security"
+---
+
+# Partnering with Microsoft Graph Security – technology partner opportunities
```
Comment (Lauragra) [inline concepts/security-partner-overview.md]: In this context, I don't think when we say "Microsoft Graph Security" that we're referring to the Security API. What exactly are we referring to? Is Security a proper noun in this context?

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[8141368](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/814136800c8ba8f031bc1391a223b9db4d6d9f1f)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 4 sec(s)_

File: concepts/security-partner-overview.md (hunk 1 above)
Comment (edwardkoval) [reply concepts/security-partner-overview.md]: I think we are referring to the API

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[8141368](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/814136800c8ba8f031bc1391a223b9db4d6d9f1f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/edwardkoval-RSA2019-NewTopicPage/api-reference/beta/toc.yml) | :white_check_mark:Succeeded | |
[concepts/security-partner-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/edwardkoval-RSA2019-NewTopicPage/concepts/security-partner-overview.md) | :white_check_mark:Succeeded | |
[concepts/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/edwardkoval-RSA2019-NewTopicPage/concepts/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C2%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903020207319550-4074%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2FVY9u6aLk1gMs0kxCLnqKLpeIBaZUtDMWBcL8TMWb2Q%3D&st=2019-03-02T02%3A08%3A33Z&se=2019-04-02T02%3A13%3A33Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[b42d369](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/b42d369540e4e9dfcc9d01d86feef491bc02b3b1)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 26 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[b42d369](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/b42d369540e4e9dfcc9d01d86feef491bc02b3b1)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 4 sec(s)_

File: concepts/security-partner-overview.md (hunk 1 above)
Comment (Lauragra) [reply concepts/security-partner-overview.md]: Is an API a thing that people partner with?

File: concepts/security-partner-overview.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
+---
+title: "Partnering with Microsoft Graph Security API – technology partner opportunities "
+description: "This This article describes partnering opportunities enabled by the Microsoft Graph Security API and is designed to help product managers and business development roles understand the investment paths and provide insight into partnering value propositions. "
+localization_priority: Normal
+author: "preetikr"
+ms.prod: "security"
+---
+
+# Partnering with the Microsoft Graph Security API – technology partner opportunities
```
Comment (Lauragra) [inline concepts/security-partner-overview.md]: I updated a few places in the article to change "Microsoft Graph Security" to "Microsoft Graph Security API". In the title, I'm not sure that we need the "- technology partner opportunities" part; it seems repetitive. It still strikes me as a little odd that we're presenting it as "partnering" with the API - not using the API in their security solutions, for example. Let's think about this more next week.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[b42d369](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/b42d369540e4e9dfcc9d01d86feef491bc02b3b1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/edwardkoval-RSA2019-NewTopicPage/api-reference/beta/toc.yml) | :white_check_mark:Succeeded | |
[concepts/security-partner-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/edwardkoval-RSA2019-NewTopicPage/concepts/security-partner-overview.md) | :white_check_mark:Succeeded | |
[concepts/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/edwardkoval-RSA2019-NewTopicPage/concepts/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C2%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903020209462503-4074%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=4Ot%2BauvJ5IG0hZzHUeJ3xrhqYWUSmm6aNwBXgrbhYRY%3D&st=2019-03-02T02%3A14%3A25Z&se=2019-04-02T02%3A19%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[4757ab9](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/4757ab99e4c7c1510f827ba6d580733f8175a128)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 26 sec(s)_

File: concepts/security-partner-overview.md (hunk 1 above)
Comment (edwardkoval) [reply concepts/security-partner-overview.md]: yes, In our case we federate out the requests to our internal or external partners

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[4757ab9](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/4757ab99e4c7c1510f827ba6d580733f8175a128)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 4 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[4757ab9](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/4757ab99e4c7c1510f827ba6d580733f8175a128)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/edwardkoval-RSA2019-NewTopicPage/api-reference/beta/toc.yml) | :white_check_mark:Succeeded | |
[concepts/security-partner-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/edwardkoval-RSA2019-NewTopicPage/concepts/security-partner-overview.md) | :white_check_mark:Succeeded | |
[concepts/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/edwardkoval-RSA2019-NewTopicPage/concepts/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C2%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903020217184480-4074%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=VwCapEQeYNJ9jixvvJ%2B3lGV29ucQ6TXJphWft5xfkIQ%3D&st=2019-03-02T02%3A19%3A37Z&se=2019-04-02T02%3A24%3A37Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[7ef0b9d](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/7ef0b9db5ad65234ae1c86ce26fb01761fed7b47)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 26 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[7ef0b9d](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/7ef0b9db5ad65234ae1c86ce26fb01761fed7b47)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 4 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[7ef0b9d](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4074/commits/7ef0b9db5ad65234ae1c86ce26fb01761fed7b47)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/edwardkoval-RSA2019-NewTopicPage/api-reference/beta/toc.yml) | :white_check_mark:Succeeded | |
[concepts/security-partner-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/edwardkoval-RSA2019-NewTopicPage/concepts/security-partner-overview.md) | :white_check_mark:Succeeded | |
[concepts/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/edwardkoval-RSA2019-NewTopicPage/concepts/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C2%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903020223398607-4074%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=2GIdlRklP8EpbNlcSQCR0Kgx6LwcpIHHVZX6SEMNQxE%3D&st=2019-03-02T02%3A24%3A53Z&se=2019-04-02T02%3A29%3A53Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**